### DISCUSSION TITLE
Arduino Uno R4 WiFi LED Matrix support

### DISCUSSION THREAD

**Discussion by Frn1** [olikraus/u8g2#2248] [Q&A] [answered]
Is support for the 12x8 led matrix support planned? I mean, it's a monochrome display so...

**Comment by olikraus** [comment, marked as answer]
No. I mean I once added support for a special kind of LED matrix, but in general, the programming architecture deviates a lot from the displays. In other words, the required changes to u8g2 lib would be too much.
